signup_utils.py

```diff
@@ -1,17 +1,19 @@
 from flask import request
 import db_management
+from models.SignupResults import SignupResults
 
 
 def process_sign_up():
-    pass_text = request.form['pass']
-    ver_pass_text = request.form['ver_pass']
-    username_text = request.form['name']
+    password = request.form['pass']
```

Comment:
Same as the login issue. Don't access the request object implicitly. Pass the username, password and verification password as parameters